src/app/pages/list/status-list/status-list.component.html

```diff
@@ -2,25 +2,159 @@
     <button data-cy="toolbar-menu" *ngIf="isSmall" style="margin-left: -8px" mat-icon-button (click)="toggleMenu()">
         <mat-icon>menu</mat-icon>
     </button>
-    <div class="mat-title" [style.marginLeft.px]="isSmall ? 24 : 0">Status List</div>
+    <div class="mat-title" [style.marginLeft.px]="isSmall ? 24 : 0">Status Lists</div>
     <pxb-spacer></pxb-spacer>
+    <div class="toolbar-actions">
+        <button mat-icon-button>
+            <mat-icon style="margin: 0 8px">help</mat-icon>
+        </button>
+        <button mat-icon-button>
+            <mat-icon style="margin: 0 8px" [matBadge]="88" matBadgeColor="warn">notifications</mat-icon>
+        </button>
+        <button mat-icon-button>
+            <mat-icon>more_vert</mat-icon>
+        </button>
+    </div>
 </mat-toolbar>
-<mat-list class="alert" *ngIf="data.length > 0">
-    <pxb-info-list-item
-        *ngFor="let item of data; let index = index"
-        [statusColor]="setAlarm(item.status) ? colors.red[500] : ''"
-        [avatar]="true"
-    >
-        <mat-icon
-            pxb-icon
-            [style.color]="getIconColor(item.status)"
-            [style.backgroundColor]="item.status === 'alarm' ? colors.red[500] : ''"
-            >{{ item.icon }}</mat-icon
-        >
-        <span pxb-title>{{ item.name }}</span>
-        <span pxb-subtitle>{{ item.details }}</span>
-        <span pxb-right-content *ngIf="setAlarm(item.status)">
-            <pxb-list-item-tag [backgroundColor]="colors.red[500]" label="new"></pxb-list-item-tag>
-        </span>
-    </pxb-info-list-item>
-</mat-list>
+<section class="list-section">
+    <div class="list-example">
+        <mat-accordion class="accordion-panel">
+            <mat-expansion-panel [expanded]="true" [class.expansion-panel-mobile]="isSmall">
+                <mat-expansion-panel-header>
+                    <mat-panel-title>
+                        With Time Stamps, with Title+Subtitle+Info
+                    </mat-panel-title>
+                </mat-expansion-panel-header>
+                <mat-list class="list-section">
+                    <pxb-info-list-item
+                        *ngFor="let item of infoListItemExample"
+                        [divider]="item.id === 3 ? null : 'partial'"
+                        [statusColor]="item.statusColor"
+                        chevron="true"
+                        [avatar]="item.avatar"
+                        iconAlign="center"
+                    >
+                        <mat-icon *ngIf="item.id === 1" pxb-icon class="alarm-icon">{{ item.icon }} </mat-icon>
```

Comment:
I understand what you're going for but since nearly every item in this list has a unique element, I don't think the ListItems need to be iterable.  I would refactor so the `*ngFor="let item of infoListItemExample"` goes away and this example manually lists out each item.

Something like this:

```
<!--- item 1 -->
<pxb-info-list-item ....>
</pxb-info-list-item>


<!--- item 2 -->
<pxb-info-list-item ....>
</pxb-info-list-item>
```

As it is right now it's very difficult to read due to all the nested conditionals.

Reply:
Done. Please review.